cluster/manifests/skipper/skipper-redis.yaml

```diff
@@ -3,7 +3,7 @@ kind: StatefulSet
 metadata:
   labels:
     application: skipper-ingress-redis
-    version: v6.2.7
+    version: "7.2.4"
```

Comment:
Why not prefix with v?

Reply:
I checked other manifests and reverted to use v prefix